core/server/master/src/main/java/alluxio/master/FaultTolerantAlluxioMaster.java

```diff
@@ -88,6 +89,9 @@ public void start() throws Exception {
           stopServing();
           stopMasters();
 
+          // TODO(peis): Stop serving properly and remove this master recreation.
```

Comment:
I've removed this to stop/start more properly to avoid this master recreation. I had it initially because i thought this could fix the problem (it was recreating the masters before my journal change).